ops/services/app_service/metabase-readonly/_var.tf

```diff
@@ -0,0 +1,66 @@
+variable "name" {
+  type        = string
+  description = "App Service Name"
+}
+
+variable "env" {
+  type        = string
+  description = "Target Environment"
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "Resource Group Name"
+}
+
+variable "resource_group_location" {
+  type        = string
+  description = "App Service Location"
+}
+
+variable "app_service_plan_id" {
+  type        = string
+  description = "App Service Plan ID"
+}
+
+variable "webapp_subnet_id" {
+  type        = string
+  description = "Webapp Subnet ID"
+}
+
+variable "global_vault_id" {
+  type = string
+}
+
+variable "run_readonly_user_create" {
+  default = true
+}
+
+variable "administrator_login" {
+  default = "simple_report"
+}
+
+variable "postgres_server_name" {
```

Comment:
These Postgres vars don't necessarily need to be variables - we can make them part of `locals` unless you think we're going to need this to be parameterized.

Reply:
The only one that probably needs to be parameterized between environments is the server name, because I imagine the address might change for test vs production. However, server_name was already a variable in the existing metabase files, so I'm just reusing that.